Frameworks/CoreGraphics/CGBitmapImage.mm

```diff
@@ -444,6 +444,11 @@
     return _data->_bitsPerComponent;
 }
 
+ID2D1RenderTarget* CGBitmapImageBacking::GetRenderTarget() {
+    UNIMPLEMENTED_WITH_MSG("D2D render target is not supported with CGImageData!");
+    return nullptr;
```

Comment:
Though I have used the UNIMPLEMETED macro this is not really a stub, we will never support this

---
In reply to: [80716651](https://github.com/Microsoft/WinObjC/pull/1040#discussion_r80716651) [](ancestors = 80716651)